src/Kernel/TableKernelAleInst.cc.py

```diff
@@ -13,12 +16,14 @@
   template class TableKernel< Dim< %(ndim)s > >;
 """
 
-for Wname in ("GaussianKernel",
```

Comment:
I think this instantiation file is only for use ALE3D, correct?  Perhaps we should move this out of the Spheral source entirely into a separate thing like SpheralC, since this isn't even built by Spheral's build system so far as I can tell.

Reply:
Yes, this script is only used by ALE3D. We can put it in SpheralC if that's more appropriate.

Reply:
Yep, we should probably move this.  Not urgent right now, but in the near future.